src/domain/collaboration/whiteboard/WhiteboardVisuals/readme.md

```
@@ -0,0 +1,60 @@
+# Whiteboard Preview Image Handling
+
+## Image generation
+
+Whiteboards are exported by Excalidraw's API function [exportToCanvas](https://docs.excalidraw.com/docs/@excalidraw/excalidraw/api/utils/export).
+
+exportToCanvas receives:
+|Name | Type | Default | Description |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)